### PR TITLE
Add Show In Animation Viewer module

### LINKED ISSUE #561: View animations
I am pretty sure Archilles had this feature, this is Animation viewer. Simple as that. (The one we have in eden, where you can check all animations ingame)

### PR DESCRIPTION
**Pull request by CreepPork**
**When merged this pull request will:**
- Title
- Resolves #561

### CODE REVIEW

**Comment by CreepPork** [comment]
Achilles had a check, if the player was a virtual curator entity, but testing it in singleplayer, without the check, resulted in no issues.

File: addons/modules/stringtable.xml [hunk 1]
```diff
@@ -2444,5 +2444,8 @@
             <English>Vehicle must have other crew available</English>
             <Japanese>車両には空いている乗員席がありません</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleAnimationViewer">
+            <English>Show in Animation Viewer</English>
+        </Key>
```
Comment (mharis001) [inline addons/modules/stringtable.xml:2449]: Move this above line 2246.

**Review by mharis001 — APPROVED** [review]

**Comment by mharis001** [comment]
Should also add a docs entry before merging.

File: addons/modules/stringtable.xml [hunk 2]
```diff
@@ -2444,5 +2444,8 @@
             <English>Vehicle must have other crew available</English>
             <Japanese>車両には空いている乗員席がありません</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleAnimationViewer">
+            <English>Show in Animation Viewer</English>
```
Comment (mharis001) [inline addons/modules/stringtable.xml:2448]:
```suggestion
            <English>Show In Animation Viewer</English>
```
To match config viewer module naming.

**Review by neilzar — APPROVED** [review]

**Review by mharis001 — APPROVED** [review]

**Pull request merged**